content/en/news/releases/1.13.x/announcing-1.13.9/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Announcing Istio 1.13.9
+linktitle: 1.13.9
+subtitle: Patch Release
+description: Istio 1.13.9 patch release.
+publishdate: 2022-10-07
+release: 1.13.9
+---
+
+This release contains a patch for [CVE-2022-41715](https://github.com/golang/go/issues/55949) and bug fixes to improve robustness. This release note describes what’s different between Istio 1.13.8 and Istio 1.13.9.
+
+{{< relnote >}}
+
+## Changes
+
+- **Fixed** an issue where user can not delete `iop` resource with revision if `istiod` is not running.  ([Issue #40796](https://github.com/istio/istio/issues/40796))
+
+- **Fixed** a bug where the return dynamically generated by `jwks` was not base64 encoded, causing Envoy to fail to parse it.
+
+- **Fixed** an issue where a root namespace `Sidecar` configuration would be ignored.
+
+## Security update
```

Comment:
I would think that this section should go above the changes.

Reply:
Done.